samples/cpp/benchmark_app/statistics_report.cpp

```diff
@@ -113,35 +147,170 @@ void StatisticsReport::dump_performance_counters(const std::vector<PerformaceCou
             dump_performance_counters_request(dumper, pc);
         }
     } else if (_config.report_type == averageCntReport) {
-        auto getAveragePerformanceCounters = [&perfCounts]() {
-            std::vector<ov::ProfilingInfo> performanceCountersAvg;
-            // iterate over each processed infer request and handle its PM data
-            for (size_t i = 0; i < perfCounts.size(); i++) {
-                // iterate over each layer from sorted vector and add required PM data
-                // to the per-layer maps
-                for (const auto& pm : perfCounts[i]) {
-                    int idx = 0;
-                    for (; idx < performanceCountersAvg.size(); idx++) {
-                        if (performanceCountersAvg[idx].node_name == pm.node_name) {
-                            performanceCountersAvg[idx].real_time += pm.real_time;
-                            performanceCountersAvg[idx].cpu_time += pm.cpu_time;
-                            break;
-                        }
-                    }
-                    if (idx == performanceCountersAvg.size()) {
-                        performanceCountersAvg.push_back(pm);
-                    }
-                }
-            }
-            for (auto& pm : performanceCountersAvg) {
-                pm.real_time /= perfCounts.size();
-                pm.cpu_time /= perfCounts.size();
-            }
-            return performanceCountersAvg;
-        };
-        dump_performance_counters_request(dumper, getAveragePerformanceCounters());
+        dump_performance_counters_request(dumper, get_average_performance_counters(perfCounts));
     } else {
         throw std::logic_error("PM data can only be collected for average or detailed report types");
     }
     slog::info << "Performance counters report is stored to " << dumper.getFilename() << slog::endl;
 }
+
+void StatisticsReportJSON::dump_parameters(nlohmann::json& js, const StatisticsReport::Parameters& parameters) {
+    for (auto& parameter : parameters) {
+        parameter.write_to_json(js);
+    }
+};
+
+void StatisticsReportJSON::dump() {
+    nlohmann::json js;
+    std::string name = _config.report_folder + _separator + "benchmark_report.json";
+
+    dump_parameters(js["cmd_options"], _parameters.at(Category::COMMAND_LINE_PARAMETERS));
+    dump_parameters(js["configuration_setup"], _parameters.at(Category::RUNTIME_CONFIG));
+    dump_parameters(js["execution_results"], _parameters.at(Category::EXECUTION_RESULTS));
+    dump_parameters(js["execution_results"], _parameters.at(Category::EXECUTION_RESULTS_GROUPPED));
```

Comment:
This call should be under the [same](https://github.com/openvinotoolkit/openvino/blob/f45991bd64fd47cb38898355557177d0a68461ab/samples/cpp/benchmark_app/main.cpp#L967) condition:
`if (FLAGS_pcseq && app_inputs_info.size() > 1)`
Right now in case of inference only mode, without group statictics it causes error:
```
[ ERROR ] map::at
terminate called after throwing an instance of 'std::out_of_range'
  what():  map::at
```